synapseclient/client.py

```diff
@@ -2747,7 +2748,24 @@ def _download_from_URL(
                 )
                 break
             elif scheme == "ftp":
-                urllib_request.urlretrieve(url, destination)
+                transfer_start_time = time.time()
+
+                def _ftp_report_hook(
```

Comment:
Nit: does this hook already exist as a convenience function? we use this logging function in other places already.

Reply:
Makes sense to me.  Thanks!